README.md

```diff
@@ -6,136 +6,147 @@ to run the subject(s).
 
 ### Installation
 
+In order to run this software via a container, you will need to acquire a copy of the FreeSurfer License for yourself. 
+
+[Follow this link for a FreeSurfer License](https://surfer.nmr.mgh.harvard.edu/fswiki/License)
+
 #### Using Docker
 
-You will need to load the image onto your docker service by running the following command:
+Before running, you will need to load the image onto your Docker service by running the following command:
 
 ```{bash}
 docker pull dcanlabs/abcd-hcp-pipeline
 ```
 
-It is common to receive a "no space left on device" error during this pull process. You may need to clean up any old/dangling images and containers from the docker registry, and possibly increase the amount of space allocated to Docker.
+If you receive a "no space left on device" error during this pull process.
+You may need to clean up any old/dangling images and containers from the docker registry, and possibly increase the amount of space allocated to Docker.
```

Comment:
start with 'y' instead of 'Y'. Also might want to tell them they have to contact ACC for this.